python/ray/tests/test_metrics_agent.py

```diff
@@ -180,18 +180,20 @@ def _setup_cluster_for_test(request, ray_start_cluster):
 
     worker_should_exit = SignalActor.remote()
 
+    extra_tags = {"ray_version": ray.__version__}
+
     # Generate a metric in the driver.
     counter = Counter("test_driver_counter", description="desc")
-    counter.inc()
+    counter.inc(tags=extra_tags)
 
     # Generate some metrics from actor & tasks.
     @ray.remote
     def f():
         counter = Counter("test_counter", description="desc")
         counter.inc()
         counter = ray.get(ray.put(counter))  # Test serialization.
-        counter.inc()
-        counter.inc(2)
+        counter.inc(tags=extra_tags)
+        counter.inc(2, tags=extra_tags)
```

Comment:
are these verified somewhere in the test?

Reply:
We validate that the counter is present

Reply:
@alexeykudinkin can you handle ^ and lmk?  I will merge the PR after this